src/test/java/tests/RepositorySelectorPopulationTests.java

```diff
@@ -47,16 +49,17 @@ public void noJsonFiles() {
     public void oneInvalidJsonFile() throws IOException {
         File invalidJson = new File("store/test/dummy-dummy.json");
         assert invalidJson.createNewFile();
-        RepoIO testIO = new RepoIO(true, true);
+        RepoIO testIO = TestController.createTestingRepoIO(Optional.of(new JSONStoreStub()));
```

Comment:
I think `noJsonFiles()` can use the stub, but `oneInvalidJsonFile()` should stay using proper JSONStore. Will that still affect the coverage?